teamB/JeongWooJin/.gitignore

```
@@ -0,0 +1,4 @@
+.env
+node_modules/
+db/schma.sql
```

```suggestion
db/schema.sql
```
우진님-! 오타가 있어 git이 계속해서 schema.sql 파일을 tracking하고 push 하고 있네요! 

수정 후 아래와 같이 따라해주세요.

- .gitignore에 등록하기 전에 한번이라도 git으로 관리해버린 파일들은(add, commit 등) 계속해서 git에 의해서 추적이 됩니다.
- 이를 없애기 위해서는 git의 기존 캐쉬를 삭제해줘야합니다.
- 아래의 명령어를 통해 캐쉬 삭제해주시고 git으로 추적 안되는 것 확인해주세요
- `git rm --cached 파일이름`
- 리모트의 git cache 삭제 후에는 반드시 리모트와 동기화 해줘야 합니다.
- `git add .`
- `git commit -m “커밋메시지”`
- `git push origin 브랜치이름`